hw5/Task_6.ts

```diff
@@ -0,0 +1,17 @@
+//Task 6
+// Напишите функцию **fooboo** которая принимает в качестве аргумента три параметра:
+// + булевое значение
+// + функцию **foo** которая выводит в консоль свое имя
+// + функцию **boo** которая выводит в консоль свое имя
+// Если переданное булевое значение **true** запускаем функцию **foo** иначе **boo**
+
+function fooBoo (myBool: boolean): boolean {
```

Comment:
You need to create 3 functions. And in 3rd one do something like
function fooBoo (myBool: boolean, foo: () => void, boo: () => void )

Reply:
I created 3 functions as you advised. Still doesn't work :(